content/en/docs/tasks/security/authentication/auto-mtls/index.md

```diff
@@ -1,379 +0,0 @@
----
-title: Automatic mutual TLS
-description: A simplified workflow to adopt mutual TLS with minimal configuration overhead.
-weight: 10
-keywords: [security,mtls,ux]
----
-
-This tasks shows a simplified workflow for mutual TLS adoption.
-
-With Istio auto mutual TLS feature, you can adopt mutual TLS by only configuring authentication policy without worrying about destination rule.
-
-Istio tracks the server workloads migrated to Istio sidecar, and configures client sidecar to send mutual TLS traffic to those workloads automatically, and send plain text traffic to workloads
-without sidecars. This allows you to adopt Istio mutual TLS incrementally with minimal manual configuration.
-
-## Before you begin
-
-* Understand Istio [authentication policy](/docs/concepts/security/#authentication-policies) and related
-[mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication) concepts.
-
-* Install Istio with the `global.mtls.enabled` option set to false and `global.mtls.auto` set to true.
-For example, using the `demo` configuration profile:
-
-{{< text bash >}}
-$ istioctl manifest apply --set profile=demo \
-  --set values.global.mtls.auto=true \
-  --set values.global.mtls.enabled=false
-{{< /text >}}
-
-## Instructions
-
-### Setup
-
-Our examples deploy `httpbin` service into three namespaces, `full`, `partial`, and `legacy`.
```

Comment:
auto mtls is not invisible to them.

when they are using workload selector beta policy or headless services.

how do we explain them this behavior differences if we never introduce auto mTLS concept to them?

Reply:
I think we need a task or faq somewhere to describe what to do. Having a separate auto-mTLS task doesn't seem right to me, and the existing content doesn't high light such scenario neither.

In the long run, we do want to make auto-mTLS invisible to user as much as possible, so I think simplify the tasks is better for users.